RollGen.Domain/DomainDice.cs

```diff
@@ -49,7 +49,7 @@ public string ReplaceWrappedExpressions<T>(string source, string expressionOpen
                 var lenientReplacedDice = ReplaceDiceExpression(matchGroupValue, true);
                 var unevaluatedMatch = Evaluate(lenientReplacedDice);
                 var target = expressionOpen + matchGroupValue + expressionClose;
-                var replacement = BooleanOrType<T>(unevaluatedMatch).ToString();
+                var replacement = Utils.BooleanOrType<T>(unevaluatedMatch).ToString();
```

Comment:
Then yes, they should stay public. :smile: I would prefer still if we made it injectable instead of static, and wrapped these functions in unit tests.  Also, we should add integration tests that verify that a boolean expression does evaluate correctly.

If we make it injectable, the interface can be public (in RollGen project), but the implementation can be internal, conforming with the standards of forcing use of Ninject to acquire implementations.